src/TSMapEditor/Models/EditorConfig.cs

```diff
@@ -168,7 +168,11 @@ private void ReadScriptActions()
                 var scriptSection = iniFile.GetSection(sections[i]);
                 scriptAction.ReadIniSection(scriptSection);
 
-                ScriptActions.Add(scriptAction.Index, scriptAction);
+                if (ScriptActions.ContainsKey(scriptAction.ID))
+                    throw new INIConfigException(
+                        $"Error while adding Script Action {scriptAction.Name}: a Script Action with ID {scriptAction.ID} already exists!");
```

Comment:
```suggestion
                if (ScriptActions.ContainsKey(scriptAction.ID))
                {
                    throw new INIConfigException($"Error while adding Script Action {scriptAction.Name}: " + 
                        $"a Script Action with ID {scriptAction.ID} already exists!");
                }
```

As stated in the contribution guide

> Braceless code block bodies should be made only when both code block head and body are single line. Statements that split into multiple lines and nested braceless blocks are not allowed within braceless blocks

Also I think splitting the string just looks easier to read, since the string itself is so long.